electrumx/server/session.py

```diff
@@ -1695,14 +1695,24 @@ async def address_status(self, hashX):
         db_history, cost = await self.session_mgr.limited_history(hashX)
         mempool = await self.mempool.transaction_summaries(hashX)
 
-        status = ''.join(f'{hash_to_hex_str(tx_hash)}:'
-                         f'{height:d}:'
-                         f'{VaultTxType(int.from_bytes(tx_type, "big")).name:s}:'
-                         for tx_hash, height, tx_type in db_history)
-        status += ''.join(f'{hash_to_hex_str(tx.hash)}:'
-                          f'{-tx.has_unconfirmed_inputs:d}:'
-                          f'{VaultTxType(tx.type).name:s}:'
-                          for tx in mempool)
+        if self.protocol_tuple < (2, 0):
```

Comment:
Maybe it's worth extracting (2,0) to a constant?

Reply:
fixed